content/en/Reference/aws/service_modules/lambda-function.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "aws-lambda"
 linkTitle: "aws-lambda"
-date: 2021-12-02
+date: 2022-01-03
```

Comment:
So all the docs under Reference/aws are autogenerated. The real docs for these references live in the opta repo and you have to update there.

Reply:
ok, I will create a separate PR for these changes then.